blakserv/synched.h

```diff
@@ -13,6 +13,8 @@
 #ifndef _SYNCHED_H
 #define _SYNCHED_H
 
+#define DEFAULT_MOVE_INTERVAL 1000
```

Comment:
Unless this needs to be in a header visible to all source files, move it to the C file that needs it, and make it a constant (static const int DEFAULT_MOVE_INTERVAL = 1000).  Macros are gross, but in 1994 they were the accepted method.